elasticache-redis/main.tf

```diff
@@ -14,9 +14,12 @@ data "aws_subnets" "private-subnets" {
 }
 
 data "aws_caller_identity" "current" {}
+data "aws_region" "current" {}
 
 
 resource "aws_elasticache_replication_group" "redis" {
+  # checkov:skip=CKV_AWS_31:Potential cascading impact on Celery. Requires further analysis
+  # checkov:skip=CKV_AWS_191:Potential cascading impact on Celery. Requires further analysis
```

Comment:
Should we create a spike / investigation ticket?

Reply:
JIRA raised and number added to comment